doc/voq/voq_hld.md

```diff
@@ -0,0 +1,809 @@
+# VOQ SONiC
```

Comment:
Should we move all the files to the 'chassis' directory? 'voq' is a pretty non-obvious term and I think it's better for this set of docs to be under /doc/chassis than /doc/voq. What do you think?

Reply:
Let's stick to voq directory.